airgun/views/fact.py

```diff
@@ -0,0 +1,20 @@
+from widgetastic.widget import Table, Text
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView, SearchableViewMixinPF4
+
+
+class HostFactView(BaseLoggedInView, SearchableViewMixinPF4):
+    breadcrumb = BreadCrumb()
+
+    table = Table(
+        './/table',
+        column_widgets={
+            'Name': Text("./a"),
+        },
+    )
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(self.breadcrumb, exception=False)
+        return breadcrumb_loaded and self.breadcrumb.read().startswith('Facts Values ')
```

Comment:
can you please avoid the space here `'Facts Values '`

Reply:
Updated.